frontend/utils/frontend_smoke_test.go

```diff
@@ -0,0 +1,207 @@
+//go:build smoke_test
+// +build smoke_test
+
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"github.com/Azure/azure-sdk-for-go/sdk/azidentity"
+	"github.com/Azure/azure-sdk-for-go/sdk/data/azcosmos"
+	"io"
+	"net/http"
+	"os"
+	"strings"
+	"testing"
+)
+
+const (
+	apiURL           = "http://localhost:8443"
+	testSubscription = "00000000-0000-0000-0000-000000000000"
+	baseResourceID   = "/subscriptions/" + testSubscription + "?api-version=2.0"
```

Comment:
Realizing this is too specific to subscriptions, when we do clusters this wont be as universal....going to tweak this

Reply:
fixed